webrender/src/shade.rs

```diff
@@ -403,13 +404,8 @@ pub struct Shaders {
     pub cs_border_segment: LazilyCompiledShader,
 
     // Brush shaders
-    brush_solid: BrushShader,
-    brush_image: Vec<Option<BrushShader>>,
-    brush_blend: BrushShader,
-    brush_mix_blend: BrushShader,
-    brush_yuv_image: Vec<Option<BrushShader>>,
-    brush_radial_gradient: BrushShader,
-    brush_linear_gradient: BrushShader,
+    brush: BrushShaders,
+    brush_debug_overdraw: BrushShaders,
```

Comment:
a brush shader struct contains 3 versions:
```rust
struct BrushShader {
    opaque: LazilyCompiledShader,
    alpha: LazilyCompiledShader,
    dual_source: Option<LazilyCompiledShader>,
}
```
Would it make sense to add a 4th for the debug overdraw instead of duplicating the whole thing?